Xamarin.Forms.Core/Layout.cs

```diff
@@ -237,6 +237,13 @@ protected void UpdateChildrenLayout()
 			double w = Math.Max(0, width - Padding.HorizontalThickness);
 			double h = Math.Max(0, height - Padding.VerticalThickness);
 
+			var isHeadless = CompressedLayout.GetIsHeadless(this);
+			var headlessOffset = CompressedLayout.GetHeadlessOffset(this);
+			for (var i = 0; i < LogicalChildrenInternal.Count; i++) {
```

Comment:
Braces on their own lines. You're in Core territory, now! muwahaha

Reply:
don't scratch that itch! :)

fixed by removing the braces.